tribler.sh

```diff
@@ -27,7 +27,8 @@ if [ "$UNAME" = "Linux" ]; then
     }
 
     python2.7 $TRIBLER_SCRIPT
-
+elif [ ! -z `uname -s | grep CYGWIN_NT*` ]; then
```

Comment:
Remove the *, that means any amount of T's at the end (or will expand to a file name if there's any matching that name on $PWD.